dask_expr/_expr.py

```diff
@@ -1041,10 +1042,37 @@ def _simplify_up(self, parent):
 
 
 class RenameSeries(Elemwise):
-    _parameters = ["frame", "index"]
-    _keyword_only = ["index"]
+    _parameters = ["frame", "index", "sorted_index"]
+    _keyword_only = ["index", "sorted_index"]
+    _defaults = {"sorted_index": False}
     operation = M.rename
 
+    @functools.cached_property
+    def _meta(self):
+        return make_meta(
+            meta_nonempty(self.frame._meta).rename(index=self.operand("index"))
+        )
+
+    @property
+    def _kwargs(self) -> dict:
+        return {"index": self.operand("index")}
+
+    def _divisions(self):
+        index = self.operand("index")
+        if is_scalar(index) and not isinstance(index, Callable):
+            return self.frame.divisions
+        elif self.sorted_index and self.frame.known_divisions:
+            old = pd.Series(1, index=self.frame.divisions)
+            new_divisions = old.rename(index).index
+            if not new_divisions.is_monotonic_increasing:
+                raise ValueError(
+                    "The renamer creates an Index with non-monotonic divisions. "
```

Comment:
```suggestion
                    "The rename creates an Index with non-monotonic divisions. "
```

Reply:
No that was intentional (it might not be proper English though?), renamer as the renaming function

Reply:
ah, okay. I thought it could also work like that. :+1: